SetMeta.Tests/Impl/OptionValueConverterTestFixture.cs

```diff
@@ -42,5 +113,75 @@ public void GetStringValue_WhenFormatProviderIsNull_ThrowException()
 
             Assert.That(ex.ParamName, Is.EqualTo("formatProvider"));
         }
+
+        [Test]
+        public void GetStringValueRuCulture_WhenWePassNull_NullShouldBeReturned()
+        {
+            Assert.DoesNotThrow(() =>
+            {
+                var actual = Sut.GetStringValue(null, new CultureInfo("ru-RU"));
+
+                Assert.IsNull(actual);
+            });
+        }
+
+        [Test]
+        public void GetStringValueRuCulture_WhenWePassEmptyString_EmptyStringShouldBeReturned()
+        {
+            Assert.DoesNotThrow(() =>
+            {
+                var actual = Sut.GetStringValue("", new CultureInfo("ru-RU"));
```

Comment:
new CultureInfo("ru-RU")
Change with private method, which returns this new value, same for "en-US" and in all places